com.unity.render-pipelines.universal/Runtime/Passes/CopyColorPass.cs

```diff
@@ -77,30 +65,33 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             CommandBuffer cmd = CommandBufferPool.Get();
             using (new ProfilingScope(cmd, ProfilingSampler.Get(URPProfileId.CopyColor)))
             {
-                RenderTargetIdentifier opaqueColorRT = destination.Identifier();
-
-                ScriptableRenderer.SetRenderTarget(cmd, opaqueColorRT, BuiltinRenderTextureType.CameraTarget, clearFlag,
+                ScriptableRenderer.SetRenderTarget(cmd, destination ?? k_CameraTarget, k_CameraTarget, clearFlag,
                     clearColor);
 
                 bool useDrawProceduleBlit = renderingData.cameraData.xr.enabled;
                 switch (m_DownsamplingMethod)
                 {
                     case Downsampling.None:
-                        RenderingUtils.Blit(cmd, source, opaqueColorRT, m_CopyColorMaterial, 0, useDrawProceduleBlit);
+                        RenderingUtils.Blit(cmd, source, destination ?? k_CameraTarget, m_CopyColorMaterial, 0, useDrawProceduleBlit);
```

Comment:
Not sure why destination would be null here. But it seems the logic is to bind the camera target in case destination is null, you can get camera color target by using renderingData.cameraData.renderer.cameraColorTexture.